### ISSUE TITLE
Make Breadcrumbs keyboard-accessible

### ISSUE THREAD

**Issue by Nifdee** [palantir/blueprint#4916]
Extracting from https://github.com/palantir/blueprint/issues/3925 which flags a couple of issues.

#### Environment

- __Package version(s)__:  core 3.49.1
- __Browser and OS versions__: 

#### Feature request

A breadcrumb's `onClick` action should also be executed when the enter / space key is pressed (similar to what buttons do).

**Comment by Andrewnt219** [comment]
Hi @Nifdee, I want to help with this issue.

**Comment by adidahiya** [comment]
@Andrewnt219 it's open for PRs if you want to contribute :)

**Comment by Andrewnt219** [comment]
@adidahiya This's weird. I have trouble installing node-sass for the project. My guess is that node-sass v4 depends on Node 14+, but the project's node version is limited to 12 & 13.

```bash
$ yarn verify


yarn run v1.22.17
$ npm-run-all -s compile dist:libs dist:apps -p test lint format-check
$ lerna run compile
lerna notice cli v3.22.1
lerna info versioning independent
lerna info Executing command in 14 packages: "yarn run compile"
lerna ERR! yarn run compile exited 1 in '@blueprintjs/colors'
lerna ERR! yarn run compile stdout:
$ run-p "compile:*"
$ sass-compile ./src
$ generate-css-variables --outputFileName colors _colors.scss
$ tsc -p src/
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run compile stderr:
/bin/bash: C:\blueprint\packages\colors\node_modules\.bin\..\..\..\node-build-scripts\sass-compile.sh: No such file or directory
error Command failed with exit code 127.
ERROR: "compile:css" exited with 127.
error Command failed with exit code 1.

lerna ERR! yarn run compile exited 1 in '@blueprintjs/colors'
lerna WARN complete Waiting for 3 child processes to exit. CTRL-C to exit immediately.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "compile" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by adidahiya** [comment]
@Andrewnt219 that's not a node-sass installation problem; `sass-compile.sh` is a custom bash script in our repo. It looks like you're using Windows... there may be some build issues when using Windows (like https://github.com/palantir/blueprint/issues/4221), unfortunately I haven't had a chance to test the dev environment outside of macOS / Linux...

**Comment by Andrewnt219** [comment]
@adidahiya I tried both cd inside `packages/core` and changing the method in `node_modules`. None of them is applicable anymore it seems. Guess I have to drop this because I don't have any Mac.
